### PR TITLE
Docs/add k8s deployment

### PR DESCRIPTION
**Pull request by aviveiros11**
### What's this PR do?

adds/improves documentation around deployment with Kubernetes.

### CODE REVIEW

**Comment by nllong** [comment]
is this ready for review @aviveiros11 ?

**Comment by aviveiros11** [comment]
@nllong yeah - I thought @axelstudios was going to merge after adding the windows setup!

**Comment by nllong** [comment]
gotcha, I guess we *should* support windows

File: charts/README.md [hunk 1]
```diff
@@ -0,0 +1,53 @@
...
+```
+kubectl get all
+```
+
+### For an existing deployment
+Update the web container
+```
+kubectl delete pod/web-<assigned-hash>
+```
+Note: the service will restart the pod an re-pull the image
```
Comment (nllong) [inline charts/README.md:50]:
```suggestion
Note: the service will restart the pod and re-pull the image
```

**Review by axelstudios — APPROVED** [review]

**Pull request merged**